drivers/hmis/app/views/hmis_client/services/_table.haml

```diff
@@ -0,0 +1,27 @@
+.table-responsive
+  %table.table
+    %thead
+      %tr
+        %th
+        %th Project
+        %th Entry Date
+        %th Service Date Provided
+        %th Service Name
+        %th
+    %tbody
+      - custom_services.each do |service|
+        %tr
+          %td
+            .ds{class: "ds-color-#{service.data_source_id}"}= ds_short_name_for(service.client.id)
+          %td= service.project.name
+          %td= service.enrollment.entry_date
+          %td= service.date_provided
+          %td= service.service_type.custom_service_category.name == service.service_name ? service.service_name : "#{service.service_type.custom_service_category.name} - #{service.service_name}"
+          %td.text-right
+            - if @client.hmis_source_visible_by?(current_user)
+              = link_to 'HMIS', source_datum_path(service.enrollment.id, type: 'Enrollment'), class: 'btn btn-xs btn-secondary btn-muted btn-hmis mb-2'
```

Comment:
Does this link look correct? I copied/modified from the [Income Benefit](https://github.com/greenriver/hmis-warehouse/blob/042bd5b6f97fc572ba4c4888b5e013db80ac0785/app/views/clients/rollup/_income_benefits.html.haml#L38) and [Enrollment table](https://github.com/greenriver/hmis-warehouse/blob/042bd5b6f97fc572ba4c4888b5e013db80ac0785/app/views/clients/_enrollment_table.haml#L134) examples. I was wondering if we want it to link directly to the Services page of the Enrollments dashboard.

Reply:
Hmm, so that's going to link to the enrollment.  Usually we'd link to the object in question (the service in this case) but the point of these links is generally so you can see ALL of the source data on the record.   I wonder if it would be better to just link off to the services page in the HMIS in this case.  @gigxz, thoughts on that?

Reply:
The most helpful place is probably `/client/:id/enrollments/:id/services` because that's where you can edit the service records.

Reply:
I agree that's probably the most helpful place to link to here. I'm not sure I've done the linking correctly -- I couldn't find a similar example except in the function `hmis_url_for` on the data source, so I kinda-sorta copied that. Let me know if there is a better way or somewhere else I can look to for a better example.